src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+            self.main_connector)
+
+        if flag == 0:
+            print("could not update to database")
+            return False
+
+        return True
+
+        # TODO send results to storage queue
+
+        # flag = rgutils.upload_to_queue(storage_account_name, result_file, main_connector)
+
+        # if flag != 0:
+        #    move_processed_data(result_file, 'measure_result', 'processed')
+        #    print(id[0])
+        # print(scan_measure_id[0][0])
+
+
+def main():
+    if len(sys.argv) != 3:
+        print("Please provide model_id and endpoint name.")
+        exit(1)
+
+    # destination_folder = str(sys.argv[1])
+    # db_connection_file = str(sys.argv[2])
+    # storage_account_name = str(sys.argv[3])
+    model_id = str(sys.argv[1])
+    service = str(sys.argv[2])
+    # calibration_file = str(sys.argv[6])
+    # container_name = str(sys.argv[7])
+
+    container_name = "scans"
+    destination_folder = '~'
+
+    # calibration = preprocessing.parseCalibration(calibration_file)
+    preprocessing.setWidth(int(240 * 0.75))
+    preprocessing.setHeight(int(180 * 0.75))
+
+    main_connector = dbutils.connect_to_main_database()
+
+    # TODO check if model_id is in active state
+    check_model = "select (json_metadata->>'active')::BOOLEAN from model where id = '{}';".format(model_id)
+    active = main_connector.execute(check_model, fetch_all=True)
+
+    if not active[0][0]:
+        print("model {0} is not active.... exiting".format(model_id))
+        exit(1)
+
+    select_measures = "select measure_id from artifact where not exists (SELECT measure_id from measure_result WHERE measure_id=artifact.measure_id and model_id = '{}')".format(
+        model_id) + r" and dataformat in ('pcd', 'depth') group by measure_id having count(case when substring(substring(storage_path from '_[0-9]\d\d_') from '[0-9]\d\d') in ('100', '104', '200') then 1 end) > 4 and count(case when substring(substring(storage_path from '_[0-9]\d\d_') from '[0-9]\d\d') in ('102', '110', '202') then 1 end) >4 and count(case when substring(substring(storage_path from '_[0-9]\d\d_') from '[0-9]\d\d') in ('101', '107', '201') then 1 end) > 4;"
+    measure_ids = main_connector.execute(select_measures, fetch_all=True)
+
+    replace_path = "~/" + config.ACC_NAME + '/qrcode/'
+
+    if config.ENV == "dev":
+        measure_id = [
+            ("c66050300c1ab684_measure_1601356048051_vj7fOLrU2dYwWDOT",
+             ),
+            ("c66050300c1ab684_measure_1601356093034_CFIfgb2SFufC7Pe9",
+             ),
+            ("c66050300c1ab684_measure_1601355986152_WspeDENLEBteZLRD",
+             ),
+            ("c66050300c1ab684_measure_1602060079742_LORX5MwfKQhni3gA",
+             ),
+            ("601db192d38c0816_measure_1601379417732_PRBsdWChgw8Qkoe1",
+             )]
+        for id in measure_id:
+            id_split = id[0].split('_')
+            delete_measure_result = "delete from measure_result where measure_id = '{}'".format(
```

Comment:
`delete_measure_result` sounds like a name for a function

```suggestion
            sql_stmt_deletion = "delete from measure_result where measure_id = '{}'".format(
```